infra/nebulift.infra.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The csproj file should be named in Pascal case `Nebulift.Infra` to follow conventions